### PR TITLE
refactor(dynamic-farm): using multicall from redux hooks 

### PR DESCRIPTION
**Pull request by viet-nv**

### CODE REVIEW

File: src/pages/App.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ import DarkModeQueryParamReader from 'theme/DarkModeQueryParamReader'
 import { isAddressString, isSupportLimitOrder, shortenAddress } from 'utils'
 
 import ElasticLegacyNotice from './ElasticLegacy/ElasticLegacyNotice'
+import Farm from './Farm'
```
Comment (namgold) [inline src/pages/App.tsx:44]: revert?

File: src/index.tsx [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import '@/wdyr'
```
Comment (namgold) [inline src/index.tsx:1]: why not `./wdyr` instead

Comment (viet-nv) [reply src/index.tsx:1]: lint issue, this must be on top of the file. but prettier order import will move it to the bottom

File: src/pages/App.tsx (hunk 1 above)
Comment (viet-nv) [reply src/pages/App.tsx]: fixed

**Review by namgold — DISMISSED** [review]

**Review by namgold — APPROVED** [review]

**Pull request merged**